microcloud/cmd/microcloud/main_init.go

```diff
@@ -470,9 +470,6 @@ func AddPeers(sh *service.Handler, systems map[string]InitSystem) error {
 		if err != nil {
 			return err
 		}
-
```

Comment:
Do we now require some retries with timeout? The interactive job gives `Get "http://unix.socket/1.0": dial unix /var/snap/lxd/common/lxd/unix.socket: connect: connection refused`.

Or is this unrelated to this change?